### PR TITLE
Feature/more env support

### PR DESCRIPTION
**Pull request by keatonkraiger**
Added more env variables and updated the README to include more environment variable documentation .

### CODE REVIEW

File: PWA/src/views/Timesheet.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
               </v-card-title>
               <v-card-text class="text-center subtitle-1 mt-3">
                 {{ $t('views_Timesheet_continue_desc0') }}
-                <strong> #{{ formId }}</strong><br />
```
Comment (unstable-compound) [inline PWA/src/views/Timesheet.vue:22]: Just want to verify, was this intentional?

**Review by unstable-compound — DISMISSED** [review]
Looks good to me, I think that's a really helpful description for them

**Comment by iyves** [comment]
girl im still working on this branch! w8 a HOT sec

File: PWA/package.json [hunk 2]
```diff
@@ -11,9 +11,8 @@
     "start": "node server.js",
     "test": "jest --setupFiles dotenv/config"
   },
-  "dependencies": {  
+  "dependencies": {
     "@babel/core": "^7.10.1",
-    "@vue/cli-service": "^4.4.1",
```
Comment (iyves) [inline PWA/package.json:16]: NB: this line still exists, npm just re-ordered the dependencies alphabetically.

Comment (keatonkraiger) [reply PWA/package.json:16]: weird

File: PWA/src/views/Timesheet.vue (hunk 1 above)
Comment (keatonkraiger) [reply PWA/src/views/Timesheet.vue:22]: yes, and adjusted continue existing form text 
image

**Review by iyves — APPROVED** [review]

**Pull request merged**